### PR TITLE
docs: How to launch hello world

### PR DESCRIPTION
**Pull request by ereali-aneo**
Creation of the tutorial folder 
The first Tutorial to do hello world with ArmoniK

### CODE REVIEW

**Comment by ngruelaneo** [comment]
In the actual documentation system, you need to provide the order of the markdown in the name. It is why all the file are starting by 1., 2. ... n.

File: .docs/content/2.guide/1.how-to/how-to-configure-partitions.md [hunk 1]
```diff
@@ -198,3 +198,63 @@ var createTaskReply = await submitterClient.CreateTasksAsync(
 ::alert{type="warning"}
 It is an error to set the `PartitionId` to `null`.
 ::
+
+
+## Example partition for the HelloWorld tutorial
```
Comment (ngruelaneo) [inline .docs/content/2.guide/1.how-to/how-to-configure-partitions.md:203]:
```suggestion
## Add a partition for the HelloWorld worker
```

File: .docs/content/2.guide/2.tutorial/HelloWorld.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+   Open the .Net file in [csharp/native/HelloWorld/Client/.](./csharp/native/HelloWorld/Client/.) as a project.
+
+   Open the Debug property and add as arguments using the command line:
+
+   ```args
+   --endpoint "http://<ip>:5001" --partition <partition name>
+   ```
+
+   Run the project.
```
Comment (ngruelaneo) [inline .docs/content/2.guide/2.tutorial/HelloWorld.md:52]: how? si click sur bouton, rajouter une image

File: .docs/content/2.guide/2.tutorial/HelloWorld.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Tutorial : HelloWorld with ArmoniK 
+
+The goal of this tutorial is to learn how to use ArmoniK by doing a HelloWorld.
```
[the remark this answers is not in the document]
Comment (ereali-aneo) [reply .docs/content/2.guide/2.tutorial/HelloWorld.md]: I believe the folder and file names were more of an issue than this sentence; I modified them in the last commit for  'how-to-launch-HelloWorld-Sample'

File: .docs/content/2.guide/2.tutorial/HelloWorld.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Tutorial : HelloWorld with ArmoniK 
+
+The goal of this tutorial is to learn how to use ArmoniK by doing a HelloWorld.
+
+You need to git clone [ArmoniK](https://github.com/aneoconsulting/ArmoniK/tree/main) and [ArmoniK.Samples](https://github.com/aneoconsulting/ArmoniK.Samples) for this tutorial.
```
[the remark this answers is not in the document]
Comment (ereali-aneo) [reply .docs/content/2.guide/2.tutorial/HelloWorld.md]: I applied the suggested change.

File: .docs/content/2.guide/2.tutorial/HelloWorld.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Copy the default one and change the partition name, add a tag to "latest", set the image to "hello".
+
+See the example [here](https://github.com/aneoconsulting/ArmoniK/blob/main/.docs/content/2.guide/1.how-to/how-to-configure-partitions.md).
+
+**Deploy** ArmoniK with `make` in [ArmoniK/infrastructure/quick-deploy/localhost](ArmoniK/infrastructure/quick-deploy/localhost).
+
+## Client
+
+   You have 3 equal ways to run the client:
```
[the remark this answers is not in the document]
Comment (ereali-aneo) [reply .docs/content/2.guide/2.tutorial/HelloWorld.md]: I applied the suggested change.

**Review by ngruelaneo — APPROVED** [review]

**Pull request merged**